controller.py

```diff
@@ -69,8 +75,14 @@ def run(self):
                 
             elif self.current_view == "criteria":
                 options = {"b":self.show_boot_message}
-
-            output = options.get(k)
-            if output is not None:
-                output()
+            
+            if options is not None:
```

Comment:
If none of the views match, `options` might not be set. Set `options = None` at the top of this loop (or, alternatively, add a final `else:` and set options to None in there.